src/sentry/api/endpoints/release_deploys.py

```diff
@@ -0,0 +1,136 @@
+            release=release,
+        )
+
+        return self.paginate(
+            request=request,
+            queryset=queryset,
+            order_by='-date_finished',
+            paginator_cls=OffsetPaginator,
+            on_results=lambda x: serialize(x, request.user),
+        )
+
+    def post(self, request, organization, version):
+        """
+        Create a Deploy
+        ```````````````
+        Create a deploy for a given release.
+        :pparam string organization_slug: the organization short name
+        :pparam string version: the version identifier of the release.
+        :param string environment: the environment you're deploying to
+        :param string name: the optional name of the deploy
+        :param url url: the optional url that points to the deploy
+        :param datetime dateStarted: an optional date that indicates when
+                                     the deploy started
+        :param datetime dateFinished: an optional date that indicates when
+                                      the deploy ended. If not provided, the
+                                      current time is used.
+
+        """
+        try:
+            release = Release.objects.get(
+                version=version,
+                organization=organization,
+            )
+        except Release.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        if not self.has_release_permission(request, organization, release):
+            raise PermissionDenied
+
+        serializer = DeploySerializer(data=request.DATA)
+
+        if serializer.is_valid():
+            result = serializer.object
+            try:
+                env = Environment.objects.get(
+                    organization_id=organization.id,
+                    name=result['environment'],
+                )
+            except Environment.DoesNotExist:
+                env = Environment.objects.create(
+                    organization_id=organization.id,
+                    name=result['environment'],
+                )
+
+            try:
+                with transaction.atomic():
+                    deploy, created = Deploy.objects.create(
+                        organization_id=organization.id,
+                        release=release,
+                        environment_id=env.id,
+                        date_finished=result.get('dateFinished', datetime.datetime.utcnow()),
+                        date_started=result.get('dateStarted'),
+                        name=result.get('name'),
+                        url=result.get('url'),
+                    ), True
+            except IntegrityError:
+                deploy, created = Deploy.objects.get(
+                    organization_id=organization.id,
+                    release=release,
+                    environment_id=env.id,
+                ), False
+                deploy.update(
+                    date_finished=result.get('dateFinished', datetime.datetime.utcnow()),
```

Comment:
Use `timezone.now()` instead. I'm not entirely sure the differences, if any, but it's what we use elsewhere and makes me think less. `from django.utils import timezone; timezone.now()`